statik.go

```diff
@@ -185,20 +185,20 @@ func loadTemplate(name string, path string, buf *string) (tmpl *template.Templat
 	return
 }
 
-func isDir(path string) (err error) {
+func requireDir(path string) (err error) {
 	dir, err := os.Stat(path)
 	if err != nil {
 		return err
 	}
 	if !dir.IsDir() {
-		return fmt.Errorf("Expected %s to be a directory", path)
+		return fmt.Errorf("expected %s to be a directory", path)
 	}
 	return nil
 }
 
 // The input path dir is assumed to be already absolute
-func newFile(info os.FileInfo, dir string) (fz FuzzyFile, f File, err error) {
-	if info.IsDir() {
+func newFile(entry os.DirEntry, dir string) (fz FuzzyFile, f File, err error) {
```

Comment:
C'è un motivo per usare una `DirEntry`, in una funzione che lavora su un file oltretutto?

Reply:
Eliminando `io/util` sono costretto ad usare questa, perchè dove viene chiamata l'iteratore mi ritorna un `DirEntry`.

Reply:
kk